doc/makefile-variables.txt

```diff
@@ -107,3 +107,40 @@ that need to be applied to the test output for just this component.
   have been found which have required altering this to "-xO3".  There are also
   studio_OPT.$(MACH).$(BITS) versions of this available if greater specificity
   is needed.
+
+If you frequently rebuild the same code, such as when you maintain a build server
+or iterate recipes for the same component, you can benefit by caching the build
+products with "ccache" - so for repeated input conditions you'd get same output
+objects quickly.
+
+Being experimental, the "ccache" integration in OI-Userland build system is
+disabled by default. See below for hints how to enable and optionally configure it.
+
+* To enable the "ccache" integration in OI-Userland build system you must
+  `export ENABLE_CCACHE=true` in the calling shell, or set the make-variable
+  like this: `gmake ENABLE_CCACHE=true build`
+* CCACHE_DISABLE is an inverted toggle of the "ccache" program, which our common
+  shared-macros.mk Makefile also honours. By original project's intent, if you
+  execute "ccache" then you want it to do its job. If you are debugging it or
+  unsure if it causes problems, you can easily temporarily disable the "ccache"
+  logic by `export CCACHE_DISABLE=true` so it quickly calls the original compiler.
```

Comment:
No. "CCACHE_DISABLE" is from ccache program. It is outside our control, and it has the inverted semantics for reasons already told. Even if we don't want to abuse the other project's variable, we can not break it into what we deem prettier.